toolchain/lex/tokenized_buffer.cpp

```diff
@@ -271,77 +267,72 @@ class TokenizedBuffer::Lexer {
     buffer_->token_infos_.back().has_trailing_space = true;
   }
 
-  auto SkipWhitespace(llvm::StringRef& source_text) -> bool {
-    const char* const whitespace_start = source_text.begin();
+  auto LexHorizWhitespace(llvm::StringRef& source_text) -> void {
```

Comment:
I don't think abbreviating "Horizontal" here is worthwhile, and is a bit jarring given that you don't abbreviate "Vertical" below.

Reply:
Agreed, will fix before landing. Had meant to switch this but forgot, thanks for catching it.